x/ccv/consumer/keeper/keeper.go

```diff
@@ -211,6 +215,57 @@ func (k Keeper) DeletePendingChanges(ctx sdk.Context) {
 	store.Delete(types.PendingChangesKey())
 }
 
+func (k Keeper) LastSovereignHeight(ctx sdk.Context) int64 {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.LastSovereignHeightKey())
+	if bz == nil {
+		return 0
+	}
+	height := sdk.BigEndianToUint64(bz)
+	return int64(height)
+}
+
+func (k Keeper) SetLastSovereignHeight(ctx sdk.Context, height int64) {
+	bz := sdk.Uint64ToBigEndian(uint64(height))
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.LastSovereignHeightKey(), bz)
+}
+
+func (k Keeper) IsPreCCV(ctx sdk.Context) bool {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.PreCCVKey())
+	if bz != nil {
+		return true
+	}
+	return false
+}
+
+func (k Keeper) SetPreCCV(ctx sdk.Context, state *types.GenesisState) {
```

Comment:
We usually make quick unit tests for CRUD operations like these, especially because this method is not just setting a flag